### PR TITLE
Load GL commands from libhwgl.so

### PR DESCRIPTION
**Pull request by dsrbecky**
Related to #355

This significantly improves the situation (we can trace and replay).

However many functions still can not be patched (seems likely all just because they are "not-implemented" stubs, so not really essential).

A few valid functions (e.g. glScissor) fail to trace due to use of PC-relative instructions, which is not implemented yet.

### CODE REVIEW

File: gapii/cc/android/installer.cpp [hunk 1]
```diff
@@ -197,7 +198,19 @@ void Installer::install_gles() {
         const char* name = gapii::kGLESExports[i].mName;
         void* func_export = gapii::kGLESExports[i].mFunc;
         bool import_found = false;
-        for (int i = 0; i < NELEM(gDriverPaths); ++i) {
+        if (drivers[0] != nullptr) { // libhwgl.so
+            // Huawei implements all functions in this library with prefix,
+            // all GL functions in libGLES*.so are just trampolines to his.
```
Comment (ben-clayton) [inline gapii/cc/android/installer.cpp]: Does this also include EGL functions?

Comment (dsrbecky) [reply gapii/cc/android/installer.cpp]: I am not sure.  Should I find out?

Comment (ben-clayton) [reply gapii/cc/android/installer.cpp]: Not really important, I just thought you said it did.

**Comment by ben-clayton** [comment]
> A few valid functions (e.g. glScissor) fail to trace due to use of PC-relative instructions, which is not implemented yet.

So we still can't fully trace on this device?

**Comment by dsrbecky** [comment]
I would say that for practical purposes it should mostly work, but still want to see if I can do better...

**Comment by ben-clayton** [comment]
`glScissor` seems like a fairly important function to trace. I'm not sure we can remove it from the milestone just yet.

**Pull request merged**